network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -416,10 +421,103 @@ class Meta:
+        self.template = "about/press.html"
+        return self.serve(request)
+
+    @route(r'^about/contact/$', name='contact-view')
+    def about_contact_page(self, request):
+        self.template = "about/contact.html"
+        return self.serve(request)
+
+    @route(r'^about/methodology/$', name='methodology-view')
+    def about_methodology_page(self, request):
+        self.template = "about/methodology.html"
+        return self.serve(request)
+
+    @route(r'^about/meets-minimum-security-standards/$', name='min-security-view')
+    def about_mss_page(self, request):
+        self.template = "about/minimum_security.html"
+        return self.serve(request)
+
+    @route(r'^products/(?P<slug>[-\w\d]+)/$', name='product-view')
+    def product_view(self, request, slug):
+        # Find product by it's slug and redirect to the product page
+        # If no product is found, redirect to the BuyersGuide page
+        try:
+            product = ProductPage.objects.get(slug=slug, live=True)
+            redirect_url = product.url
+        except ProductPage.DoesNotExist:
+            redirect_url = self.url
+        return redirect(redirect_url)
+
+    @route(r'^categories/(?P<slug>[\w\W]+)/', name='category-view')
+    def categories_page(self, request, slug):
+        # If getting by slug fails, also try to get it by name.
+        try:
+            category = BuyersGuideProductCategory.objects.get(slug=slug)
+        except BuyersGuideProductCategory.DoesNotExist:
+            category = get_object_or_404(BuyersGuideProductCategory, name__iexact=slug)
+
+        # TODO
+        # def sort_on_creepiness(product_set):
+        #     return sorted(product_set, key=get_average_creepiness)
+        #
+        # Must sort pages by their creepiness levels
+        # key = f'products_category__{slug.replace(" ", "_")}'
+        # products = cache.get_or_set(
+        #     key,
+        #     lambda: sort_on_creepiness(
+        #         [p.to_dict() for p in Product.objects.filter(product_category__in=[category]).distinct()]
+        #     ),
+        #     86400
+        # )
+        products = []
+
+        # def filter_draft_products(request, products):
+        #     if request.user.is_authenticated:
+        #         return products
+
+        #     return filter(lambda p: p['draft'] is False, products)
+
+        # products = filter_draft_products(request, products)
+
+        return render(request, 'category_page.html', {
+            'pagetype': 'category',
+            'categories': BuyersGuideProductCategory.objects.all(),
+            'category': category,
+            'products': products,
+            'mediaUrl': MEDIA_URL,
+            'pageTitle': pgettext(
+                'This can be localized. This is a reference to the “*batteries not included” mention on toys.',
+                '*privacy not included') + f' - {category}',
+        })
+
+    def get_sitemap_urls(self, request):
+        # TODO add category URLs
```

Comment:
There's a ticket for this already – I just wanted to make sure this was stubbed out early.